backend/internal/handlers/classrooms/assignments/works/works.go

```diff
@@ -77,6 +77,22 @@ func (s *WorkService) getWorksInAssignment() fiber.Handler {
 			return err
 		}
 
+		// update works with current github data (this should be put into a periodic job eventually)
+		for _, work := range works {
+			opts := &github.CommitsListOptions{
+				ListOptions: github.ListOptions{
+					PerPage: 100,
+				},
+			}
+			commits, err := s.appClient.ListCommits(c.Context(), work.OrgName, work.RepoName, opts)
+			if err != nil {
+				return errs.GithubAPIError(err)
+			}
+			work.CommitAmount = len(commits)
+			work.LastCommitDate = commits[0].GetCommit().GetCommitter().Date
+			work.FirstCommitDate = commits[len(commits)-1].GetCommit().GetCommitter().Date
+		}
+
```

Comment:
This does not work. It still shows N/A for individual students, and the overall commit score does not increment properly. Please move this into its own ticket and handle it there.